packages/server/src/routes/cron/notifyOnUnminted.ts

```diff
@@ -89,11 +90,16 @@ export const notifyOnUnminted = (
             notificationHistoryCollection.doc(member.id)
           );
           if (await shouldNotify(auth, member, memberNotificationHistory)) {
-            const title = `Mint ${calculateMaxMintableForMember(member)
-              .round(2, 0)
-              .toString()} Raha before it's too late!`;
-            const body =
-              "Your mintable amount will be capped at 40 Raha on November 15th. Tap this notification to mint now and avoid losing Raha!";
+            const isMintCapped = isPastMintCapTransitionDate();
+            const maxMintable = calculateMaxMintableForMember(
+              member
+            ).toString();
+            const title = isMintCapped
+              ? `Mint ${maxMintable} Raha now!`
+              : `Mint ${maxMintable} Raha before it's too late!`;
+            const body = isMintCapped
+              ? "You can't get any more basic income until you've minted. Tap to mint now and keep receiving your Raha!"
```

Comment:
Future work: I'm thinking some people might not want to be notified regularly when they hit the max amount, especially while Raha has no actual value. We should allow users to manage these in in-app Settings and via system-level notification settings (https://developer.android.com/guide/topics/ui/notifiers/notifications Notification Channels on Oreo) so people don't block all notifications from Raha.

Reply:
Agreed. Created #502 to track.